CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 <!-- towncrier release notes start -->
 
+## [0.38.0](https://github.com/Substra/substra-documentation/releases/tag/0.38.0) - 2024-06-03
+
+### Added
+
+- Documentation on how to use the Harbor profile on substra-backend ([#416](https://github.com/Substra/substra-documentation/pull/416))
```

Comment:
I am concerned there is not more added in the documentation in the light of the release notes. Did we not agree to document the NetworkPolicy part for instance? (cc @guilhem-barthes)

Reply:
I added an entry about PSS that was not included in the changelog FYI.

Reply:
Thank you. This should not block the release anyway, but I think we need to create follow-up issues.